src/encoded/static/scss/encoded/modules/_search.scss

```diff
@@ -149,6 +149,83 @@ $search-results-above-results-row-height: 40px;
 	}
 }
 
+/**********************************************************
+			In Experiment Sets Table (Item Pages)
+			// Carry/re-use table cell styles over to ItemPageTable (to be deprecated).
+**********************************************************/
+
+
+.item-page-table-container {
+	
+	overflow-x: auto;
+
+	.item-page-table-row-container {
+		position: relative;
+		border-bottom: 1px solid #eee;
+
+		> .inner-wrapper {
+			padding-left: 0;
+			border-top: 1px solid #eee;
+			padding-left: 47px;
+			@include generic-results-flex-description-container;
+			@include generic-results-item-detail;
+			
+		}
+
+		.table-row {
+			.column {
+				@include search-result-column-block;
+				float: none;
+				display: inline-block;
+				vertical-align: middle;
+				&:first-child {
```

Comment:
Updated to :first-child from [data-first-visible-column] as latter is not case for ItemPageTable. Was typo made when `[data-first-visible-column]` was first introduced (and copied-pasted into here, when shouldn't have been).